keps/sig-cluster-lifecycle/kubeadm/0029-20180918-kubeadm-phases-beta.md

```diff
@@ -87,6 +88,9 @@ duplication of code or in some case inconsistencies between the init and phase i
 - As a kubernetes administrator/IT automation tool, I want to run all the phases of
   the `kubeadm init` workflow except some phases.
 - Same as above, but for the `kubeadm join` workflow
+- As a kubernetes administrator/IT automation tool, I used kubeadm init/join and
+  excluded some phases, I want to run all the phases of kubeadm upgrade except
+  for some phases.
```

Comment:
Maybe 
```
- As a kubernetes administrator/IT automation tool, I want to run only one or some phases
  of the `kubeadm upgrade` workflow.
```

Reply:
We should probably emphasize, that these phases are usually equivalent to the ones already excluded during init/join.

Reply:
Will do.